src/Services/PatientService.php

```diff
@@ -224,7 +224,7 @@ public function getAll($search)
                    ethnicity
                 FROM patient_data";
 
-        if ($search['name'] || $search['fname'] || $search['lname'] || $search['dob']) {
+        if ($search['name'] || $search['dob'] || $search['city'] || $search['state'] || $search['postal_code'] || $search['phone_contact'] || $search['address'] || $search['sex'] || $search['country_code']) {
```

Comment:
I gotta tell ya's, this drives me nuts! Nothing for you to do, i'm just saying....